test/metabase/sync/analyze/fingerprint/fingerprinters_test.clj

```diff
@@ -11,14 +11,32 @@
          (transduce identity
                     (fingerprinter (field/map->FieldInstance {:base_type :type/DateTime}))
                     [#t "2013" nil #t "2018" nil nil #t "2015"])))
+  (testing "handle ChronoLocalDateTime"
+    (is (= {:global {:distinct-count 2
+                     :nil%           0.0}
+            :type   {:type/DateTime {:earliest "2013-01-01T20:04:00Z"
+                                     :latest   "2018-01-01T04:04:00Z"}}}
+           (transduce identity
+                      (fingerprinter (field/map->FieldInstance {:base_type :type/Temporal}))
+                      [(java.time.LocalDateTime/of 2013 01 01 20 04 0 0)
```

Comment:
It would be good to test the code paths from numbers and strings, not just directly inserting the problematic types.